test/core/end2end/fixtures/http_proxy_fixture.cc

```diff
@@ -521,7 +521,11 @@ grpc_end2end_http_proxy* grpc_end2end_http_proxy_create(
   grpc_core::ExecCtx exec_ctx;
   grpc_end2end_http_proxy* proxy =
       static_cast<grpc_end2end_http_proxy*>(gpr_malloc(sizeof(*proxy)));
-  memset(proxy, 0, sizeof(*proxy));
+  proxy->proxy_name = nullptr;
```

Comment:
Instead, I suggest initializing these in the struct, and then changing the code to use `grpc_core::New<>` and `grpc_core::Delete<>`.